SmartDeviceLink/SDLFileManager.m

```diff
@@ -204,6 +209,10 @@ - (void)sdl_listRemoteFilesWithCompletionHandler:(SDLFileManagerListFilesComplet
     __weak typeof(self) weakSelf = self;
     SDLListFilesOperation *listOperation = [[SDLListFilesOperation alloc] initWithConnectionManager:self.connectionManager completionHandler:^(BOOL success, NSUInteger bytesAvailable, NSArray<NSString *> *_Nonnull fileNames, NSError *_Nullable error) {
         if (error != nil || !success) {
+            if([error.localizedDescription isEqualToString:@"DISALLOWED"]) {
+                [weakSelf.mutableRemoteFileNames addObjectsFromArray:fileNames];
+                weakSelf.bytesAvailable = bytesAvailable;
```

Comment:
Why are you setting this data here?

Reply:
because if we don't and we let the setup continue bytesAvailable will be 0

Reply:
It'll still be 0 because a failed RPC will either return 0 or nil.

Reply:
removed